custom_code/mmt.py

```diff
@@ -103,7 +103,7 @@ def observation_payload(self):
             'objectid': re.sub('[^a-zA-Z0-9]', '', target.name),  # only alphanumeric characters allowed
             'ra': ra,
             'dec': dec,
-            'epoch': 2000,
+            'epoch': J2000,
```

Comment:
Is this right, that Binospec requires the integer 2000, but MMIRS requires a string J2000? If so, maybe we should update the `mmtapi` module so that they match.

Reply:
I checked this and looks like for Binospec 2000 seems to fill in the epoch tab, however, for MMIRS you need J2000.